examples/fadion/factorial.py

```diff
@@ -1,43 +1,44 @@
 def rec_factorial(x):
-    if isinstance(x, int) and x > -1:
```

Comment:
A co jakby rozbić to na dwa sprawdzenia warunku?
``` python
if not isinstance(x, int):
    return False

if x < 0:
    return False
```

Czy takie rozbicie nie dodaje czytelności ?